src/Microsoft.AspNetCore.JsonPatch/Internal/JsonPatchErrorHelper.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.JsonPatch.Exceptions;
+
+namespace Microsoft.AspNetCore.JsonPatch.Internal
+{
+    internal static class JsonPatchErrorHelper
```

Comment:
We need to report error in 3 classes, so moved it out a helper, so any suggestions or ok to keep it like this?